### PR TITLE
Format fix 2013/mills/README.md

### PR DESCRIPTION
**Pull request by xexyl**

### CODE REVIEW

**Comment by xexyl** [comment]
Hmm .. this one has greyed out resolve conflicts ... that's annoying. I saw something about that in documentation. I'll have to check again.

**Comment by xexyl** [comment]
There it allows it now.

**Comment by lcn2** [comment]
OK on this one, @xexyl:

Conflicting files
bugs.md
thanks-for-fixes.md

**Comment by xexyl** [comment]
Yes and I had just done it too I think .. let me try again!

**Comment by xexyl** [comment]
Try now please.

**Comment by xexyl** [comment]
And what an absolute mess this is! It does appear that at least for some as soon as one is merged another one which had the conflict resolved is then there again. And sometimes I have seen files where I could immediately mark them as resolved even though it claimed that that file was in conflict.

**Comment by xexyl** [comment]
I guess you had to run off .. that's okay. I got a few more things done but I'll have to do a git stash prior to doing a git pull. Then I can do a git stash pop and then do commits as necessary. This does not include many format fixes but rather more important fixes - especially to do with something that got messed up in 2001/anonymous/README.md due to adding the far superior thanks-for-fixes.md (rather than the README.md files themselves).

There are almost certainly other files with this problem that I'll be fixing later. However for now I'm done working on the repo. I hope to do more tomorrow. Off to do other things.

Hopefully merging the rest won't require a fix each time one is merged but if it does we'll just have to do it :(

**Comment by lcn2** [comment]
Yea 🎉 thanks @xexyl 

About to board a bus 🚌
